addons/l10n_in_edi/models/account_edi_format.py

```diff
@@ -434,19 +434,19 @@ def l10n_in_grouping_key_generator(base_line, tax_values):
             tags = tax_values['tax_repartition_line'].tag_ids
             line_code = "other"
             if not invl.currency_id.is_zero(tax_values['tax_amount_currency']):
-                if any(tag in tags for tag in self.env.ref("l10n_in.tax_report_line_cess").sudo().tag_ids):
+                if any(tag in tags for tag in self.env.ref("l10n_in.tax_tag_cess").sudo()):
```

Comment:
@jva-odoo It seems there was a direct link with the tax report before, so what was the reason to use separate tags instead of the expressions themselves?  

Do we still need the sudo?  It does not seem we are doing anything with it.

Reply:
We convert this report line to tags because the same tag is used in two reports.
Because most of the time we need to know the GST amounts, not in one report but in all GST reports.

May we no need of Sudo (but need to check)